### PR TITLE
UBERF-5811: rework backlinks

### PR DESCRIPTION
**Pull request by kristina-fefelova**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does a local formatting is applied (rush format)
* [ ] - Does a local svelte-check is applied (rush svelte-check)
* [ ] - Does a local UI tests are executed [UI Testing](../tests/readme.md)
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

## Contributor requirements

* [ ] - Sign-off is provided. [DCO](https://github.com/apps/dco)
* [ ] - GPG Signature is provided. [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

<sub>View in Huly <a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NWU4MDllYzk1ZmRhMGNjMDhiMWI1MjEiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.AhnHgSU0elaM3vMEolmzgWYxV6dJW5XqgeGk6k9M3Sg">UBERF-5897</a></sub>

### CODE REVIEW

File: models/activity/src/index.ts [hunk 1]
```diff
@@ -117,6 +119,9 @@ export class TActivityMessage extends TAttachedDoc implements ActivityMessage {
   @Index(IndexKind.Indexed)
     lastReply?: Timestamp
 
+  @Prop(TypeBoolean(), core.string.Boolean)
```
Comment (haiodo) [inline models/activity/src/index.ts]: We need new member notification instead of hide.

File: plugins/activity-assets/lang/ru.json [hunk 2]
```diff
@@ -33,6 +33,9 @@
     "Update": "Обновить",
     "Updated": "Обновил(а)",
     "UpdatedCollection": "Обновленные",
-    "Message": "Сообщение"
+    "Message": "Сообщение",
+    "Mentioned": "Упомянул(а)",
+    "You": "Вы",
+    "Mentions": "Упомянания"
```
Comment (aonnikov) [inline plugins/activity-assets/lang/ru.json]: typo: `Упоминания`

File: plugins/document-resources/src/components/sidebar/References.svelte [hunk 3]
```diff
@@ -15,28 +15,28 @@
...
-  import chunter, { Backlink } from '@hcengineering/chunter'
   import { Ref } from '@hcengineering/core'
   import { Document } from '@hcengineering/document'
   import { createQuery } from '@hcengineering/presentation'
   import { Label, Lazy, Scroller } from '@hcengineering/ui'
+  import activity, { ActivityReference } from '@hcengineering/activity'
 
   import document from '../../plugin'
-  import BacklinkView from './BacklinkView.svelte'
+  import ReferenceView from './ReferenceView.svelte'
```
Comment (aonnikov) [inline plugins/document-resources/src/components/sidebar/References.svelte]: I wonder, can we replace this with default activity view?

File: models/activity/src/index.ts (hunk 1 above)
Comment (kristina-fefelova) [reply models/activity/src/index.ts]: Ok, now we only send `notification` about `mention`
And if text of `mention` content  was changed, then we send a new `notification`

File: plugins/activity-assets/lang/ru.json (hunk 2 above)
Comment (kristina-fefelova) [reply plugins/activity-assets/lang/ru.json]: Fixed

File: plugins/document-resources/src/components/sidebar/References.svelte (hunk 3 above)
Comment (kristina-fefelova) [reply plugins/document-resources/src/components/sidebar/References.svelte]: Yes, we can
document sidebar:
Screenshot from 2024-03-06 20-37-23

**Review by haiodo — APPROVED** [review]

**Review by aonnikov — APPROVED** [review]

**Pull request merged**